src/components/Calculator.js

```diff
@@ -1,74 +1,202 @@
+              role="button"
+              aria-hidden
+            >
+              9
+            </div>
+            <div
+              className="colored"
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              x
+            </div>
+          </div>
+          <div className="row">
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              4
+            </div>
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              5
+            </div>
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              6
+            </div>
+            <div
+              className="colored"
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              -
+            </div>
+          </div>
+          <div className="row">
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              1
+            </div>
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              2
+            </div>
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              3
+            </div>
+            <div
+              className="colored"
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              +
+            </div>
+          </div>
+          <div className="row">
+            <div
+              className="zero"
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              0
+            </div>
+            <div
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              .
+            </div>
+            <div
+              className="colored"
+              onClick={clickHandler}
+              onKeyPress={clickHandler}
+              role="button"
+              aria-hidden
+            >
+              =
             </div>
-
           </div>
-
         </div>
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
```

Comment:
- [ **OPTIONAL** ] **In the "src/App.js" file:** I think it would be better if you also update the `src/App.js` main component to a **function based** by using an **Arrow function** for `ES6` best practices.